mods/ra/mod.yaml

```diff
@@ -37,113 +37,113 @@ Packages:
 	~movies2.mix
 
 Rules:
-	./mods/ra/rules/misc.yaml
-	./mods/ra/rules/ai.yaml
-	./mods/ra/rules/player.yaml
-	./mods/ra/rules/palettes.yaml
-	./mods/ra/rules/world.yaml
-	./mods/ra/rules/defaults.yaml
-	./mods/ra/rules/vehicles.yaml
-	./mods/ra/rules/husks.yaml
-	./mods/ra/rules/structures.yaml
-	./mods/ra/rules/infantry.yaml
-	./mods/ra/rules/civilian.yaml
-	./mods/ra/rules/decoration.yaml
-	./mods/ra/rules/aircraft.yaml
-	./mods/ra/rules/ships.yaml
-	./mods/ra/rules/fakes.yaml
+	ra:rules/misc.yaml
+	ra:rules/ai.yaml
+	ra:rules/player.yaml
+	ra:rules/palettes.yaml
+	ra:rules/world.yaml
+	ra:rules/defaults.yaml
+	ra:rules/vehicles.yaml
+	ra:rules/husks.yaml
+	ra:rules/structures.yaml
+	ra:rules/infantry.yaml
+	ra:rules/civilian.yaml
+	ra:rules/decoration.yaml
+	ra:rules/aircraft.yaml
+	ra:rules/ships.yaml
+	ra:rules/fakes.yaml
 
 Sequences:
-	./mods/ra/sequences/ships.yaml
-	./mods/ra/sequences/vehicles.yaml
-	./mods/ra/sequences/structures.yaml
-	./mods/ra/sequences/infantry.yaml
-	./mods/ra/sequences/aircraft.yaml
-	./mods/ra/sequences/misc.yaml
-	./mods/ra/sequences/decorations.yaml
+	ra:sequences/ships.yaml
+	ra:sequences/vehicles.yaml
+	ra:sequences/structures.yaml
+	ra:sequences/infantry.yaml
+	ra:sequences/aircraft.yaml
+	ra:sequences/misc.yaml
+	ra:sequences/decorations.yaml
 
 TileSets:
-	./mods/ra/tilesets/snow.yaml
-	./mods/ra/tilesets/interior.yaml
-	./mods/ra/tilesets/temperat.yaml
-	./mods/ra/tilesets/desert.yaml
+	ra:tilesets/snow.yaml
+	ra:tilesets/interior.yaml
+	ra:tilesets/temperat.yaml
+	ra:tilesets/desert.yaml
 
 Cursors:
-	./mods/ra/cursors.yaml
+	ra:cursors.yaml
 
 Chrome:
-	./mods/ra/chrome.yaml
+	ra:chrome.yaml
 
 Assemblies:
 	./mods/common/OpenRA.Mods.Common.dll
```

Comment:
I haven't tried it, but I assumed since that is pretty much an engine thing it doesn't need to be moved around anyway, so I just left it with the full path.